aten/src/ATen/core/Type.h

```diff
@@ -45,64 +45,14 @@ struct Quantizer;
 static inline void noop_deleter(void*) {}
 
 enum class TypeID {
```

Comment:
does it match TensorTypeID now precisely?

Reply:
No, there are some TensorTypeIDs that are unique to caffe2.